### PR TITLE
Prevent panic on queue package loading

### PR DESCRIPTION
**Pull request by Ferlonas**

### CODE REVIEW

File: backend/queue/rmq.go [hunk 1]
```diff
@@ -16,7 +16,10 @@ var (
 	queueHealthLimits sync.Map
 )
 
-func init() {
+func initDefault() {
+	if rmqConnection != nil {
```
Comment (daemonfire300) [inline backend/queue/rmq.go:20]: Singleton pattern?

Comment (daemonfire300) [reply backend/queue/rmq.go:20]: Let me rephrase that: Am I seeing a singleton pattern here, and if yes is this intentional, and if yes why?

Comment (Ferlonas) [reply backend/queue/rmq.go:20]: As only one connection to redis is required for the queue to work, I intended to do this on package loading. Unfortunately, this crashes the unit tests, as the init function is called even if the package is not used, only imported.

I'm switching to this way of doing it as I don't want to have the caller call the initialization first, and I do not want to wrap it in another struct, which I would then create with NewConnection(), whose only purpose would be to initialize this stuff.

Comment (daemonfire300) [reply backend/queue/rmq.go:20]: Is there a functionality to "close" the queue and does that also close the connection to redis?

Comment (Ferlonas) [reply backend/queue/rmq.go:20]: There are ways to close the queue, but as a connection creates multiple queues, this will not close the connection itself.

Comment (daemonfire300) [reply backend/queue/rmq.go:20]: Ok, so I am accepting this, but I do not have the best of feelings because I suspect we might run into problems reusing the same redis connection for every queue if this should ever be used with multiple queues and multiple routines (multiple redis servers).
